extensions/xslt/deployment/src/main/java/org/apache/camel/quarkus/component/xslt/deployment/XsltProcessor.java

```diff
@@ -149,4 +150,10 @@ void xsltResources(
         }
     }
 
+    //It should be removed with fixing https://github.com/apache/camel-quarkus/issues/1615
+    @BuildStep
+    void runtimeInit(BuildProducer<RuntimeInitializedClassBuildItem> deps) {
+        deps.produce(new RuntimeInitializedClassBuildItem("org.apache.hc.client5.http.impl.auth.NTLMEngineImpl"));
```

Comment:
Please note that since you have only one `RuntimeInitializedClassBuildItem` to produce, you could omit the producer and just return your instance of `RuntimeInitializedClassBuildItem`, something like:
```suggestion
    RuntimeInitializedClassBuildItem runtimeInit() {
        return new RuntimeInitializedClassBuildItem("org.apache.hc.client5.http.impl.auth.NTLMEngineImpl");
```

Reply:
@essobedo Thanks for the suggestion and it looks good to me.